websites/A/atomicradio/presence.ts

```diff
@@ -39,6 +110,27 @@ function clearPresenceData() {
 }
 
 presence.on("UpdateData", async () => {
+  if (window.location.origin === "https://status.atomicradio.eu") {
+    clearPresenceData();
+    presenceData.details = "Viewing the current status";
+    presenceData.startTimestamp = browsingTimestamp;
+    presence.setActivity(presenceData, true);
+    return;
+  }
+
+  if (window.location.origin === "https://docs.atomicradio.eu") {
+    clearPresenceData();
+    presenceData.details = "Viewing the documentation";
+    presenceData.startTimestamp = browsingTimestamp;
+    presence.setActivity(presenceData, true);
+    return;
+  }
+
+  if (window.location.origin !== "https://atomicradio.eu") {
+    clearPresenceData();
+    return;
+  }
```

Comment:
I don't see sense in this if statement because you only have 2 subdomains and one main domain.